runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBoxOverlay.java

```diff
@@ -122,4 +122,10 @@ public Dimension render(Graphics2D graphics)
 
 		return panel.render(graphics);
 	}
+
```

Comment:
this seems like this should be a fixup for 15a83c719bb9d52cf84550f3471543cafc6261c5?